celsos_bookSearch/celsos_bookSearch/Services/GetService.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  GetService.swift
+//  celsos_bookSearch
+//
+//  Created by Celso Junio Simões de Oliveira Santos on 21/05/21.
+//
+
+import Foundation
+import PromiseKit
+
+//class GetService
+// TO DO
+// Formato tirado de um POST. GET a construir
+class GetService {
+
+    static var baseURL = ""
+
+    func postLogin(_ emailText: String, _ passwordText: String) {
+
+        guard let url = URL(string: "\(GetService.baseURL)login") else { return }
+        var request = URLRequest(url: url)
+        request.httpMethod = "GET"
+        request.setValue("application/json", forHTTPHeaderField: "Content-Type")
+        request.setValue("application/json", forHTTPHeaderField: "Accept")
+        let usuario = Book()
+
+        guard let jsonData = try? JSONEncoder().encode(usuario) else { return }
+        request.httpBody = jsonData
+
+        let task = URLSession.shared.dataTask(with: request) { data, response, error in
+            guard let data = data,
```

Comment:
_data_ is not used. Can be replaced by _ for example

Reply:
Sure. Actually, this entire class was inserted there just for reference. It was not the initial intention to built the service code inside this branch too. So, i will not really use this messy of mine. Sorry about that